osidb/tests/test_flaw.py

```diff
@@ -1194,3 +1196,84 @@ def test_flaw_affects_rhscl_invalid_collection(
         affect = AffectFactory(ps_module=ps_module, ps_component=ps_component)
         if alerts:
             assert set(alerts).issubset(affect._alerts)
+
+    @pytest.mark.parametrize(
+        "comment_attrs,should_raise",
+        [
+            ([("This issue has been addressed\n\n\n\nVia", "False")], True),
```

Comment:
This is wrong, `"False"` should just be a boolean, that's why tests are failing and you shouldn't actually need `ast.literal_eval` since you're storing a scalar value